modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,614 @@
+
+NOTE: If `xhtml` standard representation is required, explicitly setting `element_format` to `xhtml` works as before.
+
+=== `image_upload_handler`
+
+Previously the `images_upload_handler` option passed `success` or `failure` callbacks to report the status of an image upload. In {productname} 6, `images_upload_handler` reports upload results by returning a `Promise`, with a value of either `resolve` or `reject`.
+
+=== `link_default_protocol`
+
+The `link_default_protocol` option sets the link protocol used by links added or edited using the _link_ dialog.
+
+It now defaults to `https`. Previously it defaulted to `http`.
+
+NOTE: The `link_default_protocol` value is only applied to an edited or inserted link if the protocol is not explicitly specified.
+
+[[text-patterns]]
+== Text Patterns
+
+As noted in the xref:6.0-release-notes.adoc#removed-or-deprecated-options-textpattern[_{productname} 6.0 Release Notes_], the `textpattern_patterns` option:
+
+* was renamed to `text_patterns`;
+* and had its functionality moved to the Core of {productname} 6.0.
+
+As well, the `textpattern` API has been removed.
+
+Finally, and unlike in previous versions of {productname}, text patterns are now on by default. `text_patterns: false` turns the functionality off.
+
+NOTE: Using `text_patterns: []` to turn `text_patterns` off is not supported. It sets `text_patterns` to an empty array. Having, consequently, no patterns to match, it presents as if `text_patterns` is off. Appearance is not reality, however. Setting `text_patterns: false` is the only supported way of disabling `text_patterns`.
+
+After upgrading:
+
+* rename any options in your {productname} init configuration to match the new name.
+* remove `textpattern` from your plugins list.
+
+=== tinymce.activeEditor.options.get
+
+Previously `tinymce.activeEditor.options.get('plugins')` returned the configured plugins as a string.
+
+As of {productname} 6, `tinymce.activeEditor.options.get('plugins')` now returns the configured plugins as an array.
+
+
+[[plugins]]
+== Plugins
+
+=== Plugin loading format change
 
-[[editor-core]]
-== Editor-Core
+{productname} 6.0 allows plugin loading in any of three formats:
 
-To migrate the core TinyMCE editor to TinyMCE 6, review the following sections:
+. An array of strings:
++
+`plugins: [ 'advlist', 'list', 'image', 'help', 'wordcount' ]`
++
+. A space-separated string:
++
+`plugins: 'advlist list image help wordcount'`
++
+. A comma-separated string:
++
+`plugins: 'advlist,list,image,help,wordcount'`
 
-* xref:cloud-delivery[Cloud Delivery].
-* xref:option-setting-changes[Option/setting changes].
-* xref:plugin-changes[Plugin Changes]
+Previously, two other formats were allowed: comma-separated strings with spaces and arrays of space-separated strings.
```

Comment:
Per my previous comment, it's only the latter that was removed and the former wasn't ever explicitly supported (though should still work).
```suggestion
Previously, one other format was allowed: arrays of space-separated strings.
```